tests/system/tests/bash/enable_plugin_coap.bash

```diff
@@ -60,15 +71,11 @@ if [[ "${COAP_PLUGIN}" == "" ]]
 then
     echo "COAP plugin is not already activated, enabling - |${COAP_PLUGIN}|"
 
-    curl -k -s -S -X POST http://${FOGLAMP_SERVER}:${FOGLAMP_PORT}/foglamp/service -d '{ "name"   : "coap", "type"   : "south", "plugin" : "coap"}' | jq -S "."
-
     export SCHEDULE_ID=` curl -k -s -S -X GET http://${FOGLAMP_SERVER}:${FOGLAMP_PORT}/foglamp/schedule | jq --raw-output '.schedules | .[] | select(.processName=="coap") | .id'`
 
     echo SCHEDULE_ID -${SCHEDULE_ID}-
 
-    curl -k -s -S -X PUT http://${FOGLAMP_SERVER}:${FOGLAMP_PORT}/foglamp/schedule/${SCHEDULE_ID} -d '{ "enabled" : true}'  | jq -S "."
-
-    $TEST_BASEDIR/bash/sleep.bash 10
+    curl -k -s -S -X GET http://${FOGLAMP_SERVER}:${FOGLAMP_PORT}/foglamp/schedule/${SCHEDULE_ID}   | jq -S "."
```

Comment:
What is the purpose of this curl? We are neither storing the response anywhere nor validating..

Reply:
this type of commands are used to feed monitoring files, like for example : 

- 0025_prepare_plugins_out.temp
- 0025_prepare_plugins_err.temp

to have information in case of troubleshooting.